### PR TITLE
Refactor: migrate to latest version

### PR DESCRIPTION
**Pull request by axe312ger**

### CODE REVIEW

File: gatsby/README.md [hunk 1]
```diff
@@ -5,95 +5,40 @@
...
-    Use the Gatsby CLI to create a new site, specifying the default starter.
+Set up environment variables. We recommend [direnv](https://direnv.net/) but you might want to create `.env.*` files as suggested by the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/)
 
-    ```shell
-    # create a new Gatsby site using the default starter
-    gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default
-    ```
+```
+export GATSBY_MATOMO_URL="https://your-matomo-instance.com"
+export GATSBY_MATOMO_SITE_ID=42
```
Comment (zcei) [inline gatsby/README.md:25]: Is this strictly necessary to run, even if you don't want to check tracking calls in Matomo?

File: gatsby/src/components/consent-manager/wrapper.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { youTubeIntegration } from "@techboi/consent-manager-integration-youtube"
+
+const consentManagerConfig = {
+  integrations: [
+    matomoIntegration({
+      matomoURL: process.env.GATSBY_MATOMO_URL,
+      siteID: process.env.GATSBY_MATOMO_SITE_ID,
+    }),
+    youTubeIntegration(),
+    {
```
Comment (zcei) [inline gatsby/src/components/consent-manager/wrapper.js:18]: Sweet to demonstrate inline integration definition 🎉

**Review by zcei — APPROVED** [review]

File: gatsby/src/components/consent-manager/wrapper.js (hunk 2 above)
Comment (axe312ger) [reply gatsby/src/components/consent-manager/wrapper.js:18]: 🙇

File: gatsby/README.md (hunk 1 above)
Comment (axe312ger) [reply gatsby/README.md:25]: It won't break the site but will result in 404 when tracking is called

**Pull request merged**